doc/source/morph_stats.rst

```diff
@@ -112,8 +111,13 @@ are to be split. This is a sample output using the above configuration:
 For more information on the application and available options, invoke it with the ``--help``
 or ``-h`` option.
 
-
 .. code-block:: bash
 
     morph_stats --help
 
+Features
+********
+
+.. runblock:: console
+
+    $ morph_stats -l
```

Comment:
sadly the autorun plugin doesn't allow me to hid the commands executed when using the pycon driver, so I have to use the console one.